RLTest/__main__.py

```diff
@@ -721,6 +739,10 @@ def run_jobs(jobs, results, port):
                 for failure in failures:
                     print('\t\t' + failure)
             sys.exit(1)
+        else:
+            if self.args.failed_tests_file:
+                with open(self.args.failed_tests_file, 'w') as file:
+                    file.write('')
```

Comment:
Did you mean to write empty here? If so, either of the following are common. WDYT

```
fp = open(self.args.failed_tests_file, 'w')
fp.close()
```

or

```
with open(self.args.failed_tests_file, 'w') as file:
    pass
```

Reply:
Although opened with 'w', it seems that if nothing is being written to the file, it's not being overwritten.